src/js/components/award/shared/overview/ProgressBar.jsx

```diff
@@ -0,0 +1,560 @@
+/**
+ * ProgressBar.jsx
+ * Created by Jonathan Hill 11/26/19
```

Comment:
Don't be lyin.

Reply:
lol, that is when i started it, with the first iteration of the new design of the dates section